.github/workflows/compile-queries.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Setup CodeQL
         uses: ./.github/actions/fetch-codeql
         with:
-          channel: 'release'
+          channel: 'nightly'
```

Comment:
Is this what we want? We need to make sure the latest from main is still compilable by the latest release. With this change, we can miss a breaking change in the queries.

Reply:
Changed to suspend compile checking of this query in the meantime